app/views/agent/nextPaymentDue/paymentDue.scala.html

```diff
@@ -151,8 +174,8 @@ <h2 id="over-due-payments-heading" class="heading-large">@messages("agent.paymen
                     <table class="govuk-table-whatYouOwe">
                         @tableHead("over-due-payments-table-head")
                         <tbody>
-                            @for((charge, index) <- chargesList.overduePaymentList.zipWithIndex) {
-                            @tableRow(s"over-due-type-$index", charge, isOverduePayment = true)
+                            @for((charge, index) <- chargesList.sortedOverduePaymentLists.zipWithIndex) {
+                            @tableRow2(s"over-due-type-$index", charge, isOverduePayment = true, latePaymentCharge = true)
```

Comment:
Why is latePaymentCharge always set to true? This is the only place the method is called so what does it do?